src/api/app/controllers/webui/user_controller.rb

```diff
@@ -52,16 +52,17 @@ def do_login
   end
 
   def show
+    limit = (params[:length] || 25).to_i
```

Comment:
Yes and no. The table on that page, that uses the queried data, allows to set the table page lenght ([here](https://github.com/bgeuken/open-build-service/blob/4c4250d94b337fc2527a5b35d914df484d69d7ab/src/api/app/controllers/webui/user_controller.rb#L101)).
So I added it to that action. Just in case that someone uses that parameter. We could actually hardcode it to 25.